tests/integration/test_vulnerability_detector/test_feeds/alas/test_extra_tags_alas_feed.py

```diff
@@ -1,6 +1,56 @@
-# Copyright (C) 2015-2021, Wazuh Inc.
-# Created by Wazuh, Inc. <info@wazuh.com>.
-# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+'''
+copyright: Copyright (C) 2015-2021, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: Set of tests that check the behavior of Vulnerability Detector when there is an extra tag in the feed.
+
+tier: 2
+
+modules:
+    - vulnerability_detector
+
+components:
+    - manager
+
+daemons:
+    - wazuh-modulesd
+    - wazuh-db
+    - wazuh-analysisd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - CentOS 6
+    - Ubuntu Focal
+    - Ubuntu Bionic
+    - Ubuntu Xenial
+    - Ubuntu Trusty
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 8
+    - Red Hat 7
+    - Red Hat 6
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/vulnerability-detection/
+
+tags:
+    - vulnerability
+    - vulnerability_detector
```

Comment:
The tags `feeds` should be added
```suggestion
    - vulnerability
    - vulnerability_detector
    - feeds
```

Reply:
Done in this [link](https://github.com/wazuh/wazuh-qa/commit/c02bcd31ed6d3d0f57d519bdc2051b8eafeede60#:~:text=c02bcd31ed6d3d0f57d519bdc2051b8eafeede60)